calcrule_social_protection/converters/string_to_python_values.py

```diff
@@ -0,0 +1,9 @@
+import ast
+
+
+def convert_to_python_value(string):
```

Comment:
I wonder if this should be moved to utils.py of this module or moved to the core module. Folder 'converters' is for building objects mostly.